src/desktop/apps/artwork/test/fixtures/acquireable_artwork.json

```diff
@@ -4,7 +4,7 @@
       "_id": "56e86588b202a366da000571",
       "id": "yee-wong-exploding-powder-movement-blue-and-pink",
       "is_acquireable": true,
-      "is_inquireable": false,
+      "is_inquireable": true,
```

Comment:
Artworks that are `acquireable` tend to also be `inquireable`. This makes our tests a little more robust.